### PR TITLE
Added telugu translations to the proj

### PR DESCRIPTION
**Pull request by moulibheemaneti**

### CODE REVIEW

**Comment by moulibheemaneti** [comment]
@EwenDC yeah sure.

File: public/locales/auset-te.json [hunk 1]
```diff
@@ -0,0 +1,426 @@
...
+      "setupTitle": "కోర్విడ్ కుట్రను సెటప్ చేయండి",
+      "setup": "<i>1.</i> మాతృభూమి క్లియరింగ్‌ను ఎంచుకోండి. మీకు నచ్చిన 1 <strong>యోధుడు</strong> మరియు 1 <strong>ప్లాట్ టోకెన్</strong>ని అక్కడ ఉంచండి.<br/><i>2.</i> 1 <strong>యోధుడిని</strong> ఉంచండి. > ప్రతి సూట్ క్లియరింగ్‌లో. <i>(మునుపటి దశను లెక్కించడం, మొత్తం 4 మంది యోధులను ఉంచండి.)</i>"
+    },
+    "cult": {
+      "name": "బల్లి కల్ట్(లిజార్డ్ కల్ట్)",
```
Comment (EwenDC) [inline public/locales/auset-te.json:170]: I think the phonetic translation is a nice touch, however if the others are removed this should also be removed for consistency.

File: public/locales/auset-te.json [hunk 2]
```diff
@@ -0,0 +1,426 @@
...
+      "setupTitle": "లిజార్డ్ కల్ట్‌ని సెటప్ చేయండి",
+      "setup": "<i>1.</i> శత్రు భూభాగాలకు ఆనుకొని లేని స్వదేశీ క్లియరింగ్‌ను ఎంచుకోండి.<br/><i>2.</i> 4 <strong>యోధులు</strong> మరియు 1 సరిపోలే <strong> ఉంచండి తోట</strong>మీ స్వదేశంలో. 3 <strong>యోధులను</strong> దాని ప్రక్కనే ఉన్న క్లియరింగ్‌లలో వీలైనంత సమానంగా ఉంచండి.<br/><i>3.</i> మీ అకోలైట్స్ బాక్స్‌లో 2 <strong>యోధులను</strong> ఉంచండి.<br /><i>4.</i> మీ గార్డెన్స్ ట్రాక్‌లను <strong>గార్డెన్స్</strong>తో పూరించండి, మ్యాప్‌లో గార్డెన్‌కి ఎడమవైపున ఉన్న స్థలం మినహా.<br/><i>5.</i> ఉంచండి మీ <strong>అవుట్‌కాస్ట్ మార్కర్</strong> అవుట్‌కాస్ట్ బాక్స్‌లోని ఏదైనా స్థలంలో దాని అవుట్‌కాస్ట్ వైపు ఉంటుంది."
+    },
+    "duchy": {
+      "name": "భూగర్భ ప్రదేశం(అండర్గ్రౌండ్ డచి)",
```
Comment (EwenDC) [inline public/locales/auset-te.json:177]: I think the phonetic translation is a nice touch, however it makes the faction name too long to fit neatly in the interface. I would suggest just using the faction name from the official Leder games translation, or if that doesn't exist use just the literal translation without the phonetic one.

File: public/locales/auset-te.json [hunk 3]
```diff
@@ -0,0 +1,426 @@
...
+      "setupTitle": "భూగర్భ డచీని ఏర్పాటు చేయండి",
+      "setup": "1 </strong> మీ స్వదేశంలో. 5 <strong>యోధులను</strong> దాని ప్రక్కనే ఉన్న క్లియరింగ్‌ల మధ్య వీలైనంత సమానంగా ఉంచండి.<br/><i>3.</i> మీ <strong>బురో బోర్డ్‌ను</strong> మ్యాప్ దగ్గర ఉంచండి. మీ <strong>సిటాడెల్స్</strong> మరియు <strong>మార్కెట్లు</strong>తో మీ భవనాల ట్రాక్‌లను పూరించండి. మీ 9 <strong>మంత్రి కార్డ్‌లను</strong> మీ అస్థిర మంత్రుల పైల్‌పై ఉంచండి. మీ 9 <strong>కిరీటాలను</strong> విజయ పాయింట్లను చూపుతున్న చతురస్రాకార ఖాళీలపై ఉంచండి."
+    },
+    "eyrie": {
+      "name": "ఐరీ రాజవంశాలు(ఐరీ డైనసైటిస్)",
```
Comment (EwenDC) [inline public/locales/auset-te.json:184]: I think the phonetic translation is a nice touch, however it makes the faction name too long to fit neatly in the interface. I would suggest just using the faction name from the official Leder games translation, or if that doesn't exist use just the literal translation without the phonetic one.

File: public/locales/auset-te.json [hunk 4]
```diff
@@ -0,0 +1,426 @@
...
+      "setup_vagabondSetUp": "<i>1.</i> మీ <strong>పాన్</strong>ని ఏదైనా అడవిలో ఉంచండి.<br/><i>2.</i> క్వెస్ట్ డెక్‌ని షఫుల్ చేయండి. సమీపంలో వ్యవహరించడానికి 3 <strong>క్వెస్ట్‌లను</strong> గీయండి.<br/><i>3.</i> అదనపు <Bag/>, <Boot/>, <Hammer/> మరియు <Sword/ని ఉంచండి > <strong>నాశన అంశాలను</strong> <i>(\"R\"తో గుర్తు పెట్టబడింది)</i> శిథిలాల కింద యాదృచ్ఛికంగా <i>(ఇప్పటికే ఉన్న వస్తువులతో)</i>.<br/><i >4.</i> \"{{vagabond}}\" <strong>అక్షర కార్డ్</strong>ని మీ క్యారెక్టర్ కార్డ్ స్లాట్‌లో ఉంచండి. <InitialStartingItems>, </InitialStartingItems> మరియు <FinalStartingItem/> <strong>ప్రారంభ అంశాలను</strong> <i>(\"S\"తో గుర్తు పెట్టబడింది)</i>ని మీ సాట్చెల్‌లో మరియు వాటి మ్యాచింగ్ ట్రాక్‌లలో ఉంచండి, అవసరం మేరకు."
+    },
+    "warlord": {
+      "name": "నూరేళ్ల స్వామి(లార్డ్ అఫ్ హుండ్రేడ్స్)",
```
Comment (EwenDC) [inline public/locales/auset-te.json:220]: I think the phonetic translation is a nice touch, however it makes the faction name too long to fit neatly in the interface. I would suggest just using the faction name from the official Leder games translation, or if that doesn't exist use just the literal translation without the phonetic one.

File: public/locales/auset-te.json [hunk 5]
```diff
@@ -0,0 +1,426 @@
...
+      "setupTitle": "రివర్‌ఫోక్ కంపెనీని ఏర్పాటు చేయండి",
+      "setup": "<i>1.</i> నది వెంబడి ఏవైనా క్లియరింగ్‌లలో 4 <strong>యోధులను</strong> ఉంచండి.<br/><i>2.</i> 3 <strong>యోధులను</strong> ఉంచండి. మీ చెల్లింపుల పెట్టె.<br/><i>3.</i> సరిపోలే <strong>ట్రేడ్ పోస్ట్‌లు</strong>తో మీ ట్రేడ్ పోస్ట్‌ల ట్రాక్‌లను పూరించండి.<br/><i>4.</i> మీ 3ని ఉంచండి. మీ సేవల ట్రాక్‌లో <strong>సేవా గుర్తులు</strong>, ప్రతి సేవకు ధరను సెట్ చేస్తుంది."
+    },
+    "vagabond": {
+      "name": "సంచారి(వాగాబాండ్)",
```
Comment (EwenDC) [inline public/locales/auset-te.json:212]: I think the phonetic translation is a nice touch, however it creates odd looking nested brackets when included in the faction select:
image

and sometimes it even causes an overflow from being too long:
image

I would suggest just using the faction name from the official Leder games translation, or if that doesn't exist use just the literal translation without the phonetic one.

File: public/locales/auset-te.json [hunk 6]
```diff
@@ -0,0 +1,426 @@
...
+      "action": "దొంగిలించు",
+      "effect": "మీ క్లియరింగ్‌లో ఉన్న ఏ ఆటగాడు నుండి అయినా యాదృచ్ఛిక కార్డ్‌ని తీసుకోవడానికి ఒక <Torch/>ని ఖాళీ చేయండి."
+    },
+    "tinker": {
+      "name": "లోహపు పాత్రలను బాగు చేయువాడు",
+      "action": "రోజు కూలీ",
+      "effect": "మీ క్లియరింగ్‌కు సరిపోయే సూట్‌ను డిస్కార్డ్ పైల్ నుండి తీసుకోవడానికి ఒక <Torch/>ని ఖాళీ చేయండి. <i>(మీరు ఎప్పుడైనా బర్డ్ కార్డ్ తీసుకోవచ్చు.)</i>"
+    },
+    "vagrant": {
+      "name": "ఒకచోట నుండి మరియొక చోటికి మారు",
```
Comment (EwenDC) [inline public/locales/auset-te.json:421]: This vagabond name is simply too long to fit in the interface. I would advise against trying to perfectly capture the meaning of the word "vagrant" and instead opt for a nice single word that could describe someone who is a trouble maker. Conciseness is far more important than accuracy here, you just want a fun word that people might associate with how this vagabond plays.

File: public/locales/auset-te.json [hunk 7]
```diff
@@ -0,0 +1,426 @@
...
+      "action": "మండుతున్న భూమి",
+      "effect": "<Torch/>ని ఖాళీ చేసి, దానిని మీ క్లియరింగ్‌లో ఉంచండి. ఆ క్లియరింగ్ నుండి అన్ని శత్రువు ముక్కలను తొలగించండి. <Torch/>తో ముక్కలను ఉంచడం లేదా క్లియరింగ్‌లోకి తరలించడం సాధ్యం కాదు. <i>(మీరు ఈ క్లియరింగ్‌లో ఉంటారు. మీరు బయటకు వెళ్లిన తర్వాత, మీరు తిరిగి లోపలికి వెళ్లలేరు. \"ఫేవర్ ఆఫ్ ది...\" కార్డ్‌తో <Torch/> తీసివేయబడదు ఎందుకంటే ఇది శత్రువు ముక్క కాదు. )</i>"
+    },
+    "thief": {
+      "name": "దొంగ",
+      "action": "దొంగిలించు",
+      "effect": "మీ క్లియరింగ్‌లో ఉన్న ఏ ఆటగాడు నుండి అయినా యాదృచ్ఛిక కార్డ్‌ని తీసుకోవడానికి ఒక <Torch/>ని ఖాళీ చేయండి."
+    },
+    "tinker": {
+      "name": "లోహపు పాత్రలను బాగు చేయువాడు",
```
Comment (EwenDC) [inline public/locales/auset-te.json:416]: Ideally the vagabond name should just be a single word that could describe someone who creates. A literal translation of "tinker" is unnecessary and not required for the gameplay. Conciseness is far more important than accuracy here, you just want a fun word that people might associate with how this vagabond plays.

File: public/locales/auset-te.json [hunk 8]
```diff
@@ -0,0 +1,426 @@
...
+      "body": "ప్రతి క్రీడాకారుడు స్కోర్ ట్రాక్ యొక్క \"0\" స్థలంలో వారి ఎంపిక చేసిన వర్గం యొక్క స్కోర్ మార్కర్‌ను ఉంచుతారు.",
+      "body_vagabondSetUp": "ప్రతి క్రీడాకారుడు స్కోర్ ట్రాక్ యొక్క \"0\" స్థలంలో వారి ఎంపిక చేసిన వర్గం యొక్క స్కోర్ మార్కర్‌ను ఉంచుతారు. ప్రతి వాగాబాండ్ ఆటగాడు రిలేషన్ షిప్‌ల ఉదాసీన ప్రదేశంలో ఆడుతున్న ప్రతి నాన్-వాగాబాండ్ వర్గానికి రిలేషన్ షిప్ మార్కర్‌ను ఉంచుతాడు."
+    },
+    "chooseHand": {
+      "title": "ప్రారంభ చేతులు ఎంచుకోండి",
+      "body": "ప్రతి క్రీడాకారుడు తమ చేతిలో మూడు కార్డ్‌లను ఉంచడానికి ఎంచుకుంటాడు మరియు మిగిలిన రెండు కార్డ్‌లను షేర్డ్ డెక్‌లో ఉంచాడు. పూర్తయిన తర్వాత, షేర్డ్ డెక్‌ని షఫుల్ చేయండి."
+    },
+    "setupEnd": {
+      "title": "ఆట ప్రారంభించండి",
+      "body": "సెటప్ ఇప్పుడు పూర్తయింది. ప్లేయర్ {{count}}తో ఆట ప్రారంభమవుతుంది. మీరు సెటప్ ప్రాసెస్‌ను పునఃప్రారంభించాలనుకుంటే, దిగువ బటన్‌ను ఉపయోగించండి.<p><i>\"రూట్ ఆటోమేటెడ్ సెటప్\" ఎవెన్ కామెరాన్ ద్వారా డెవలప్ చేయబడింది</i><br/><i>బోర్డ్ గేమ్ ఆధారంగా \"రూట్ \" Leder Games ద్వారా ప్రచురించబడింది</i><br/><i>CC BY-NC-SA 4.0</i></p> కింద లైసెన్స్ పొందిన అప్లికేషన్"
```
Comment (EwenDC) [inline public/locales/auset-te.json:125]: `<i>CC BY-NC-SA 4.0</i></p> కింద లైసెన్స్ పొందిన అప్లికేషన్`
The final line spills outside the paragraph. `</i></p>` should be _after_ the final piece of text

Also, feel free to add a credit for yourself here for work on the translation 😄. The translated line with the above fix and your credit (where it says "YOUR CREDIT HERE") should look like this:
`సెటప్ ఇప్పుడు పూర్తయింది. ప్లేయర్ {{count}}తో ఆట ప్రారంభమవుతుంది. మీరు సెటప్ ప్రాసెస్‌ను పునఃప్రారంభించాలనుకుంటే, దిగువ బటన్‌ను ఉపయోగించండి.<p><i>\"రూట్ ఆటోమేటెడ్ సెటప్\" ఎవెన్ కామెరాన్ ద్వారా డెవలప్ చేయబడింది</i><br/><i>YOUR CREDIT HERE</i><br/><i>బోర్డ్ గేమ్ ఆధారంగా \"రూట్ \" Leder Games ద్వారా ప్రచురించబడింది</i><br/><i>CC BY-NC-SA 4.0 కింద లైసెన్స్ పొందిన అప్లికేషన్</i></p>`

See the [French translation](https://github.com/EwenDC/root-automated-setup/blob/4d1e4ded732b7b5898b641c9d3fc6e4493687d50/public/locales/auset-fr.json#L125) for an example

**Review by EwenDC — CHANGES_REQUESTED** [review]
Hey, had some time to look at this. Please review the points I raised and make changes as appropriate. Thank you.

**Comment by moulibheemaneti** [comment]
Hi @EwenDC . I'll change the requested changes and I'll notify you.

**Pull request closed without merging**